src/javaRosa.js

```diff
@@ -892,6 +892,9 @@ define([
             var itextItem = widget.getItextItem();
             if (itextItem) {
                 if (widget.isDefaultLang) {
+                    if (!value) {
+                        value = widget.getPlaceholder();
+                    }
```

Comment:
Having this inside the `isDefaultLang` block seems a little wierd since it will change `value` for https://github.com/dimagi/Vellum/blob/fix-no-placeholder/src/javaRosa.js#L906, which is outside of the if-block. Does that seems like a problem to you?